src/components/Identification/index.js

```diff
@@ -24,17 +24,18 @@ import WarningIcon from '@material-ui/icons/Warning';
 import CloseIcon from '@material-ui/icons/Close';
 import Snackbar from '@material-ui/core/Snackbar';
 import IconButton from '@material-ui/core/IconButton';
-import './index.css';
-import GordonLoader from '../../../../components/Loader';
 import defaultGordonImage from './defaultGordonImage';
+import GordonLoader from '../Loader/index';
+import { windowBreakWidths } from '../../theme';
+import './index.css';
 
```

Comment:
I moved the Identification component from inside the My Profile component to "src/components". Due to this, I had to change the file paths of imported files